packages/import/src/importIssues.ts

```diff
@@ -167,14 +167,25 @@ export const importIssues = async (apiKey: string, importer: Importer): Promise<
 
   const teamInfo = await client.team(teamId);
 
-  const issueLabels = await teamInfo?.labels();
+  const teamLabels = await teamInfo?.labels();
+  const workspaceLabels = await (await client.organization).labels();
+  const existingLabels = [...(teamLabels?.nodes ?? []), ...(workspaceLabels?.nodes ?? [])];
   const workflowStates = await teamInfo?.states();
 
   const existingLabelMap = {} as { [name: string]: string };
-  for (const label of issueLabels?.nodes ?? []) {
+  const existingLabelGroupsMap = {} as { [name: string]: string };
+
+  for (const label of existingLabels) {
     const labelName = label.name?.toLowerCase();
-    if (labelName && label.id && !existingLabelMap[labelName]) {
-      existingLabelMap[labelName] = label.id;
+    const children = await label.children();
+    if (children?.nodes?.length > 0) {
+      if (labelName && label.id && !existingLabelGroupsMap[labelName]) {
+        existingLabelGroupsMap[labelName] = label.id;
```

Comment:
Should we be lowercasing `label.id` here too?

Reply:
`label.id` here is a Linear label id, so I don't think we need to lowercase it as it should always be a uuid.

Reply:
urgh, yes it's label name and that's already lower cased